tests/conftest.py

```diff
@@ -93,6 +93,17 @@ def checkout_with_item(checkout, product):
     return checkout
 
 
+@pytest.fixture
+def checkout_with_items(checkout, product_list, product):
```

Comment:
Isn't `checkout_with_item` enough for you? This fixture generates a huge amount of unneeded data and it is time and db expensive.